cartography/data/jobs/cleanup/aws_ingest_waf_v2_cleanup.json

```diff
@@ -0,0 +1,18 @@
+{
+    "statements": [{
+        "query": "MATCH (:AWSAccount{id: $AWS_ID})-[:RESOURCE]->(n:WAFv2WebACL) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n)",
```

Comment:
You have to cleanup the nodes, and then the relationships.
Do this for every node type added and for every relationship type added during your sync.